common/djangoapps/student/models.py

```diff
@@ -929,12 +929,15 @@ def is_course_full(self, course):
 
         return is_course_full
 
-    def users_enrolled_in(self, course_id):
-        """Return a queryset of User for every user enrolled in the course."""
-        return User.objects.filter(
-            courseenrollment__course_id=course_id,
-            courseenrollment__is_active=True
-        )
+    def users_enrolled_in(self, course_id, is_active=True):
+        """Return a queryset of User for every user enrolled in the course.  If is_active=None, returns both
```

Comment:
Agreed on this one, unless there is a specific product request to be able to generate a report for just unenrolled learners.